rdmd_test.d

```diff
@@ -310,6 +310,15 @@ void runTests()
         assert(res.status == -SIGSEGV, format("%s", res));
     }
 
+    /* https://issues.dlang.org/show_bug.cgi?id=11997- rdmd should search it's
```

Comment:
I know, :) but you'll have to take that up with Martin, as it's a copy-paste from his bug title.  I can still change it here if you prefer.